tests/test_askbot.py

```diff
@@ -473,6 +473,7 @@ def test_fetch(self):
         json_comments = json.loads(comments)
 
         self.assertEqual(len(questions[0]['data']['answers']), len(questions[0]['data']['answer_ids']))
+        self.assertTrue(questions[0]['data']['answers'][0]['accepted'])
```

Comment:
Don't we have any test case when this value is `False`? We should check that too.

Reply:
yes, I'm going to add it
```
self.assertFalse(questions[1]['data']['answers'][0]['accepted'])
```